ricecooker/classes/nodes.py

```diff
@@ -212,6 +216,46 @@ def get_json_tree(self):
 
         return tree
 
+
+    def save_channel_children_to_csv(self, metadata_csv, topic_structure = list()):
+        # Not including channel title in topic structure
+        
+        is_channel = isinstance(self, ChannelNode)
+ 
+        if not is_channel:
+            # Build out tag string
+            current_level = self.title
+            # prevent list from being passed by reference. Create new instance of list
+            structure_list = list(topic_structure)
+            if len(structure_list) < 1:
+                structure_list = [current_level]
+            else:
+                structure_list.append(current_level)
```

Comment:
The `if len(structure_list) < 1:` case should no longer be needed, because we're no longer using a reference to a list as the default value for the arg.

Reply:
removed this